setup.py

```diff
@@ -120,8 +120,7 @@ def build_extension(self, ext):
         build_args = ["--config", cfg, "-j", str(BUILD_JOBS)]
         subprocess.check_call(["cmake", "--build", "."] + build_args, cwd=self.build_temp)
 
-
-PACKAGE_VERSION = "0.26.0.dev14"
+PACKAGE_VERSION = "0.35.1.dev14"
```

Comment:
Since you update the base atlas version, I would reset the `dev` part to `0` (or `1` depending on your preference).